dotnetv3/Glue/README.md

```diff
@@ -22,6 +22,22 @@ minutes instead of months.
 
 ## Code examples
 
+### Single Actions
+
+- [Create a crawler](Scenarios/Glue_Basics_Scenario/Glue_Basics/GlueMethods.cs) (`CreateCrawlerAsync`)
+- [Create a database](scenarios/Glue_Basics_Scenario/Glue_Basics/GlueMethods.cs) (`CreateDatabaseAsync`)
+- [Create a job definition](Scenarios/Glue_Basics_Scenario/Glue_Basics/GlueMethods.cs) (`CreateJobAsync`)
+- [Delete a crawler](Scenarios/Glue_Basics_Scenario/Glue_Basics/GlueMethods.cs) (`DeleteCrawlerAsync`)
+- [Delete a database from the AWS Glue Data Catalog](Scenarios/Glue_Basics_Scenario/Glue_Basics/GlueMethods.cs) (`DeleteDatabaseAsync`)
+- [Delete a job definition](Scenarios/Glue_Basics_Scenario/Glue_Basics/GlueMethods.cs) (`DeleteJobAsync`)
+- [Get all jobs](Scenarios/Glue_Basics_Scenario/Glue_Basics/GlueMethods.cs) (`GetJobsAsync`)
+- [Get tables from a database](Scenarios/Glue_Basics_Scenario/Glue_Basics/GlueMethods.cs) (`GetTablesAsync`)
+- [Get runs of a job](Scenarios/Glue_Basics_Scenario/Glue_Basics/GlueMethods.cs) (`GetJobRunsAsync`)
+- [Get a crawler](Scenarios/Glue_Basics_Scenario/Glue_Basics/GlueMethods.cs) (`GetCrawlerAsync`)
+- [Get a databasse](Scenarios/Glue_Basics_Scenario/Glue_Basics/GlueMethods.cs) (`GetDatabaseAsync`)
+- [Start a crawler](Scenarios/Glue_Basics_Scenario/Glue_Basics/GlueMethods.cs) (`StartCrawlerAsync`)
+- [Start a job run](Scenarios/Glue_Basics_Scenario/Glue_Basics/GlueMethods.cs) (`StartJobRunAsync`)
```

Comment:
Most of these links are broken because the `scenarios` folder name is not capitalized but the link is: "Scenarios".
You can test this yourself before you open a PR:
1. Select the three dots in the upper right of the file.
2. Select View file. This opens the rendered version of the file.
3. Click the links to verify them.

Reply:
Fixed